client/components/Coverages/CoverageEditor/CoverageFormHeader.tsx

```diff
@@ -168,7 +175,7 @@ export class CoverageFormHeaderComponent extends React.PureComponent<IProps> {
                         </ListRow>
                     )}
                 </Column>
-                {canEditAssignment && !readOnly && (
+                {(canEditAssignment || planningConfig.planning_auto_assign_to_workflow) && !readOnly && (
```

Comment:
We have conditions set above where remove assignment action is not available (such as when in the "Add To Planning" modal in Monitoring).

Rather than using the auto assign config, why not remove the condition for `isCoverageDraft`, then it will work under the conditions outlined in the ticket and include other current business rules.

Reply:
Perfect, if that's how we judge whether the assignment has `"auto_add_to_wf"` then I'll do it